testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentOptions.java

```diff
@@ -50,6 +50,9 @@ public final class EnvironmentOptions
     @Option(names = "--debug", description = "Open Java debug ports")
     public boolean debug;
 
+    @Option(names = "--tracing", description = "Automatic collection of OpenTracing data")
+    public boolean tracing;
+
```

Comment:
Can we combine `--tracing` with `--debug` - Does tracing has its own set of use-case ?

Reply:
I wanted to avoid that. There is no need to couple Java Debugger with a tracing

Reply:
@findepi thoughts?

Reply:
Combining the two options into one should probably be avoided. I was using tracing to troubleshoot an infinite loop within the product tests. If I were not careful, the tracing container would slowly consume all memory on the computer and crash (this was without the memory limits added in this PR). I suspect that if tracing had been enabled, and I was using the debugger, then the debugger would have been unusable.

Reply:
let's keep separate

we can rename --debug to --jvm-debug if that helps (leaving old as a legacy option)

Reply:
I can rename in a follow-up PR

Reply:
(ok, but rename only if old name will cont to work)